### ISSUE TITLE
New Steam UI only opens after running `steam --reset`

### ISSUE THREAD

**Issue by ndavd** [ValveSoftware/steam-for-linux#9692]
#### Your system information

* Steam client version (build number or date): `multilib/steam 1.0.0.78-1`
* Distribution (e.g. Ubuntu): Arch
* Opted into Steam client beta?: No
* Have you checked for system updates?: Yes
* Steam Logs: [steam-logs.tar.gz](https://github.com/ValveSoftware/steam-for-linux/files/11819309/steam-logs.tar.gz)
* GPU: Nvidia

#### Please describe your issue in as much detail as possible:
Steam only opens the first time, i.e.
- Install Steam
- Open Steam
  - OK
- Reopen Steam
  - Doesn't launch
- Run `steam --reset`
- Open Steam
  - OK
- ...

I initially thought my issue was related with https://github.com/ValveSoftware/steam-for-linux/issues/9383, but now I'm a bit confused regarding of what's actually happening.

**Comment by Sajito** [comment]
Having a similar issue. Steam won't launch for me at all, even after `steam --reset`.
I can see the login screen, but after the login screen nothing happens. Only the tray icon appears, but nothing else.
Launching steam from terminal keeps printing this lines over and over again.

```
(steam:51965): GLib-GObject-CRITICAL **: 19:17:29.493: g_object_ref: assertion 'G_IS_OBJECT (object)' failed

(steam:51965): GLib-GObject-CRITICAL **: 19:17:29.493: g_object_unref: assertion 'G_IS_OBJECT (object)' failed
steamwebhelper.sh[53325]: Runtime for steamwebhelper: defaulting to /home/sajito/.local/share/Steam/ubuntu12_64/steam-runtime-heavy
steamwebhelper.sh[53325]: glibc >= 2.34, partially disabling sandbox until CEF supports clone3()

(steam:51965): GLib-CRITICAL **: 19:17:38.669: g_main_context_push_thread_default: assertion 'acquired_context' failed

(steam:51965): GLib-CRITICAL **: 19:17:38.669: g_main_context_pop_thread_default: assertion 'g_queue_peek_head (stack) == context' failed
BuildCompleteAppOverviewChange: 446 apps
RegisterForAppOverview 1: 13ms
RegisterForAppOverview 2: 13ms
```

**Your system information**

- Steam client version (build number or date): multilib/steam 1.0.0.78-1
- Distribution (e.g. Ubuntu): EndeavourOS - With KDE Plasma 5.27.6
- Opted into Steam client beta?: No
- Have you checked for system updates?: Yes
- Steam Logs: [logs.zip](https://github.com/ValveSoftware/steam-for-linux/files/11821591/logs.zip)
- GPU: Nvidia

Also running X11. Using a workstation, not a mobile computer.

**Comment by DreamingCuttlefish** [comment]
This is also happening for me. Only running --reset or appending the steam Exec line in the .desktop file with -vgui will let me launch at all

**Comment by hermestrimegistus** [comment]
> This is also happening for me. Only running --reset or appending the steam Exec line in the .desktop file with -vgui will let me launch at all

But does it let you run games? Why is nobody pointing out that even if you open steam ui  with -vgui, none of the games are opening.

**Comment by Sajito** [comment]
My games are opening fine, when using `-vgui`.
@hermestrimegistus I think you're having a different issue. Are you using mangohud? If yes, have you tried disabling/removing it?

**Comment by Candyhands** [comment]
Yep, this is the only way to open steam and actually launch games from it for me too. I'm running arch w/xfce, everything up to date. 3060ti. It's bizarre, you would think that resetting steam would fix the issue wholesale. But once you exit steam, you need to run `steam --reset`  again, and each time you want to open it. When I launch it via commandline, these are the errors that pop up.

 ```(steam:9387): GLib-GObject-CRITICAL **: 21:06:27.063: g_object_unref: assertion 'G_IS_OBJECT (object)' failed
Could not connect to X session manager: Authentication Rejected, reason : None of the authentication protocols specified are supported and h

(steam:9387): Gtk-WARNING **: 21:06:56.305: gtk_disable_setlocale() must be called before gtk_init()
RegisterForAppOverview 1: 7ms
RegisterForAppOverview 2: 7ms

(steam:9387): GLib-GObject-CRITICAL **: 21:06:56.877: g_object_ref: assertion 'G_IS_OBJECT (object)' failed

(steam:9387): GLib-GObject-CRITICAL **: 21:06:56.877: g_object_unref: assertion 'G_IS_OBJECT (object)' failed
steamwebhelper.sh[10034]: Runtime for steamwebhelper: defaulting to /home/jonah/.local/share/Steam/ubuntu12_64/steam-runtime-heavy
steamwebhelper.sh[10034]: glibc >= 2.34, partially disabling sandbox until CEF supports clone3()
BuildCompleteAppOverviewChange: 241 apps'
 ```

**Comment by hermestrimegistus** [comment]
> My games are opening fine, when using `-vgui`. @hermestrimegistus I think you're having a different issue. Are you using mangohud? If yes, have you tried disabling/removing it?

Yes my games probably have mangdohud in arguments. I uninstalled mangohud but since using -vgui is not letting me open the properties of the game I can't edit them.

**Comment by ndavd** [comment]
> > This is also happening for me. Only running --reset or appending the steam Exec line in the .desktop file with -vgui will let me launch at all
> 
> But does it let you run games? Why is nobody pointing out that even if you open steam ui  with -vgui, none of the games are opening. 

Most likely because after doing reset, it resets your game properties, I noticed that with vgui I couldn't open Properties. Whats most likely happening is that your games only run with specific launch commands and/or with a custom proton (that all resets after running steam --reset)

**Comment by hermestrimegistus** [comment]
> > > This is also happening for me. Only running --reset or appending the steam Exec line in the .desktop file with -vgui will let me launch at all
> > 
> > 
> > But does it let you run games? Why is nobody pointing out that even if you open steam ui  with -vgui, none of the games are opening.
> 
> Most likely because after doing reset, it resets your game properties, I noticed that with vgui I couldn't open Properties. Whats most likely happening is that your games only run with specific launch commands and/or with a custom proton (that all resets after running steam --reset)

Thanks friendo. steam --reset deletes all proton and other steam libraries. After setting them in compatibility the games are running as before. May god bless you!

**Comment by hermestrimegistus** [comment]
Now that a temporary fix exists. Is there a latest ticket tracking this issue and whether specific code issue has been found or not?

**Comment by Candyhands** [comment]
> > This is interesting. Doing a reinstall of xdg-desktop-portal let me open the steam client. But the same issue persists, once you close steam. You can't open it back up. But reinstalling that is much quicker than resetting steam.
> 
> Hmm odd, because I don't even have `xdg-desktop-portal` installed


I believe xdg desktop portal is a functional dependency of  steam with the latest update due to the changes to the file manager. https://bugs.archlinux.org/task/78625?project=5&string=steam

**Comment by Candyhands** [comment]
Hmm, there seems to have been an update to the client? And now reinstalling  `xdg-desktop-portal`  does nothing for me.

**Comment by ndavd** [comment]
> Hmm, there seems to have been an update to the client? And now reinstalling `xdg-desktop-portal` does nothing for me.

Maybe that was it yes.. And the worst thing is that using my workaround I can't set proton GE, because after resetting I can't install new versions of proton and if I do it after I open Steam then Steam doesn't update my current version of proton...

**Comment by barthezzv2** [comment]
I might have found some kind of way around getting it to work (on Arch KDE with nvidia 535.54.03), having it setup to autostart on login seems to highly increase the chances of it starting steam-native, no -vgui or --reset needed, xdg-desktop-portal installed, flatpak installed, everything is running as intended. If you dont see the steam splash screen when you login (basically starting steam without UI), just logout and try again, just do not close steam once it starts I guess.

One reason why I am not using the -vgui argument (despite it working fine) is that I am unable to configure the launch options for games (which is a pain on a new install).

**Comment by sanks64** [comment]
This seems to be the same issue as what I have.

---

#### Your system information

* Steam client version (build number or date): 1687386907 or Thu, Jun 22 03:18 UTC -08:00
* Distribution (e.g. Ubuntu): Arch Linux
* Opted into Steam client beta?: No
* Have you checked for system updates?: Yes
* Steam Logs: [steam-logs.tar.gz](https://github.com/ValveSoftware/steam-for-linux/files/11857389/steam-logs.tar.gz)
* GPU: Nvidia

#### Please describe your issue in as much detail as possible:

Enabling Steam Play for all other titles breaks the client UI.

Terminal logs:

```
steam.sh[183249]: Running Steam on arch rolling 64-bit
steam.sh[183249]: STEAM_RUNTIME is enabled automatically
setup.sh[183323]: Steam runtime environment up-to-date!
steam.sh[183249]: Steam client's requirements are satisfied
[2023-06-24 21:04:52] Startup - updater built Jun 21 2023 21:17:38
[2023-06-24 21:04:52] Startup - Steam Client launched with: '/home/sanks/.local/share/Steam/ubuntu12_32/steam'
06/24 21:04:52 Init: Installing breakpad exception handler for appid(steam)/version(1687386907)/tid(183378)
[2023-06-24 21:04:52] Loading cached metrics from disk (/home/sanks/.local/share/Steam/package/steam_client_metrics.bin)
[2023-06-24 21:04:52] Using the following download hosts for Public, Realm steamglobal
[2023-06-24 21:04:52] 1. https://client-update.akamai.steamstatic.com, /, Realm 'steamglobal', weight was 1000, source = 'update_hosts_cached.vdf'
[2023-06-24 21:04:52] 2. https://cdn.cloudflare.steamstatic.com, /client/, Realm 'steamglobal', weight was 1, source = 'update_hosts_cached.vdf'
[2023-06-24 21:04:52] 3. http://media.steampowered.com, /client/, Realm 'steamglobal', weight was 1, source = 'baked in'
[2023-06-24 21:04:52] Verifying installation...
[2023-06-24 21:04:52] Verification complete

(process:183378): GLib-GObject-CRITICAL **: g_object_ref: assertion `G_IS_OBJECT (object)' failed

(process:183378): GLib-GObject-CRITICAL **: g_object_unref: assertion `G_IS_OBJECT (object)' failed
Loaded SDL version 3.0.0-1782-g214d5daa3
XRRGetOutputInfo Workaround: initialized with override: 0 real: 0xf0a71370
XRRGetCrtcInfo Workaround: initialized with override: 0 real: 0xf0a6fcc0
GetWin32Stats: display was not open yet, good
GetWin32Stats: display was not open yet, good
steamwebhelper.sh[183385]: Runtime for steamwebhelper: defaulting to /home/sanks/.local/share/Steam/ubuntu12_64/steam-runtime-heavy
steamwebhelper.sh[183385]: glibc >= 2.34, partially disabling sandbox until CEF supports clone3()
Fontconfig warning: line 5: unknown element "description"
Fontconfig warning: "/etc/fonts/conf.d/10-hinting-slight.conf", line 4: unknown element "description"
Fontconfig warning: "/etc/fonts/conf.d/10-scale-bitmap-fonts.conf", line 4: unknown element "description"
Fontconfig error: "/etc/fonts/conf.d/10-scale-bitmap-fonts.conf", line 72: non-double matrix element
Fontconfig error: "/etc/fonts/conf.d/10-scale-bitmap-fonts.conf", line 72: non-double matrix element
Fontconfig warning: "/etc/fonts/conf.d/10-scale-bitmap-fonts.conf", line 80: saw unknown, expected number
Fontconfig warning: "/etc/fonts/conf.d/10-sub-pixel-rgb.conf", line 4: unknown element "description"
Fontconfig warning: "/etc/fonts/conf.d/10-yes-antialias.conf", line 4: unknown element "description"
Fontconfig warning: "/etc/fonts/conf.d/11-lcdfilter-default.conf", line 4: unknown element "description"
Fontconfig warning: "/etc/fonts/conf.d/20-unhint-small-vera.conf", line 4: unknown element "description"
Fontconfig warning: "/etc/fonts/conf.d/30-metric-aliases.conf", line 4: unknown element "description"
Fontconfig warning: "/etc/fonts/conf.d/40-nonlatin.conf", line 4: unknown element "description"
Fontconfig warning: "/etc/fonts/conf.d/45-generic.conf", line 4: unknown element "description"
Fontconfig warning: "/etc/fonts/conf.d/45-latin.conf", line 4: unknown element "description"
Fontconfig warning: "/etc/fonts/conf.d/48-spacing.conf", line 4: unknown element "description"
Fontconfig warning: "/etc/fonts/conf.d/49-sansserif.conf", line 4: unknown element "description"
Fontconfig warning: "/etc/fonts/conf.d/50-user.conf", line 4: unknown element "description"
Fontconfig warning: "/etc/fonts/conf.d/51-local.conf", line 4: unknown element "description"
Fontconfig warning: "/etc/fonts/conf.d/60-generic.conf", line 4: unknown element "description"
Fontconfig warning: "/etc/fonts/conf.d/60-latin.conf", line 4: unknown element "description"
Fontconfig warning: "/etc/fonts/conf.d/65-nonlatin.conf", line 4: unknown element "description"
CAppInfoCacheReadFromDiskThread took 68 milliseconds to initialize
Failed to init SteamVR because it isn't installed
steamwebhelper.sh[183550]: Runtime for steamwebhelper: defaulting to /home/sanks/.local/share/Steam/ubuntu12_64/steam-runtime-heavy
steamwebhelper.sh[183550]: glibc >= 2.34, partially disabling sandbox until CEF supports clone3()

(steam:183378): Gtk-WARNING **: gtk_disable_setlocale() must be called before gtk_init()
local (potentially out of sync) copy of roaming config loaded - 1337 bytes.
roaming config store loaded successfully - 1337 bytes.
migrating temporary roaming config store
BRefreshApplicationsInLibrary 1: 1ms

(steam:183378): GLib-GObject-CRITICAL **: g_object_ref: assertion `G_IS_OBJECT (object)' failed

(steam:183378): GLib-GObject-CRITICAL **: g_object_unref: assertion `G_IS_OBJECT (object)' failed
System startup time: 15.03 seconds
steamwebhelper.sh[183675]: Runtime for steamwebhelper: defaulting to /home/sanks/.local/share/Steam/ubuntu12_64/steam-runtime-heavy
steamwebhelper.sh[183675]: glibc >= 2.34, partially disabling sandbox until CEF supports clone3()
BuildCompleteAppOverviewChange: 403 apps
RegisterForAppOverview 1: 18ms
RegisterForAppOverview 2: 19ms
^Csteamwebhelper.sh[183804]: Runtime for steamwebhelper: defaulting to /home/sanks/.local/share/Steam/ubuntu12_64/steam-runtime-heavy
steamwebhelper.sh[183804]: glibc >= 2.34, partially disabling sandbox until CEF supports clone3()
BuildCompleteAppOverviewChange: 403 apps
RegisterForAppOverview 1: 10ms
RegisterForAppOverview 2: 10ms
```

#### Steps for reproducing this issue:

1. Sign in
2. Enable Steam Play for all other titles
3. Close Steam
4. Open Steam
5. The UI does not open. The tray icon is present, but trying to launch the client from it does not lead to anything.

**Comment by sanks64** [comment]
I found what fixes the issue for me!

1. Remove these:
   - `~/.local/share/Steam/`
   - `~/.steam`
   - `~/.steampath`
   - `~/.steampid`
2. Open Steam.
3. Login.
4. Go to Steam -> Settings -> Interface.
5. Disable "Enable GPU accelerated rendering in web views (requires restart)".

Never buying NVIDIA again...

**Comment by ndavd** [comment]
> I found what fixes the issue for me!
> 
> 1. Remove these:
>    
>    * `~/.local/share/Steam/`
>    * `~/.steam`
>    * `~/.steampath`
>    * `~/.steampid`
> 2. Open Steam.
> 3. Login.
> 4. Go to Steam -> Settings -> Interface.
> 5. Disable "Enable GPU accelerated rendering in web views (requires restart)".
> 
> Never buying NVIDIA again...

That actually works. Many thanks! 🚀 
PS: +1 for never buying NVIDIA again

**Comment by Sajito** [comment]
Disabling GPU acceleration doesn't work for me.
But the issue seems to only occur with X11, at least for me. When using a wayland session, the new ui is working fine.

**Comment by ndavd** [comment]
> Disabling GPU acceleration doesn't work for me. But the issue seems to only occur with X11, at least for me. When using a wayland session, the new ui is working fine.

I'm also on X11 and I actually didn't follow those steps exactly. This was what solved it for me https://github.com/ValveSoftware/steam-for-linux/issues/9383#issuecomment-1605700376

**Comment by Candyhands** [comment]
I'm on Nvidia, and enabling gpu accelerated rendering worked for me as well.

**Comment by 1MasterD1** [comment]
I'm on Nvidia with the same issue, and can confirm that solved it for me

**Comment by Sajito** [comment]
Unfortunately for me the solution does not work :(
Even after `steam --reset` the ui won't open. Started steam in a Wayland session, there it's working well. Enabled GPU acceleration, tried again in X11, but still won't start.

**Comment by joshuarcase** [comment]
On X11 with Nvidia, enabling gpu accelerated rendering fixed it for me

**Comment by Zatura** [comment]
I am using fedora 38, after migrating from Wayland/noveau to xorg/Nvidia proprietary drivers (from RPM fusion), the same is happening to me
--reset open the UI but it statys stuck at "loading my user data..".
This my journalctl sample of steam when opening without --reset
```
Jul 01 16:06:55 fedora com.valvesoftware.Steam.desktop[12704]: GetWin32Stats: display was not open yet, good
Jul 01 16:06:55 fedora com.valvesoftware.Steam.desktop[12870]: steamwebhelper.sh[181]: Runtime for steamwebhelper: defaulting to /home/main/.var/app/com.valvesoftware.Steam/.local/share/Steam/ubuntu12_64/steam-runtime-heavy
Jul 01 16:06:55 fedora com.valvesoftware.Steam.desktop[12870]: steamwebhelper.sh[181]: Running under Flatpak, disabling sandbox
Jul 01 16:06:55 fedora com.valvesoftware.Steam.desktop[12870]: steamwebhelper.sh[181]: CEF sandbox already disabled
Jul 01 16:06:55 fedora com.valvesoftware.Steam.desktop[12704]: Could not connect to X session manager: None of the authentication protocols specified are supported
Jul 01 16:06:56 fedora com.valvesoftware.Steam.desktop[12704]: CAppInfoCacheReadFromDiskThread took 137 milliseconds to initialize
Jul 01 16:06:56 fedora com.valvesoftware.Steam.desktop[12704]: Failed to init SteamVR because it isn't installed
Jul 01 16:07:28 fedora steam[12704]: gtk_disable_setlocale() must be called before gtk_init()
Jul 01 16:07:29 fedora com.valvesoftware.Steam.desktop[12704]: roaming config store loaded successfully - 231 bytes.
Jul 01 16:07:29 fedora com.valvesoftware.Steam.desktop[12704]: migrating temporary roaming config store
Jul 01 16:07:29 fedora com.valvesoftware.Steam.desktop[12704]: BRefreshApplicationsInLibrary 1: 0ms
Jul 01 16:07:29 fedora steam[12704]: g_object_ref: assertion 'G_IS_OBJECT (object)' failed
Jul 01 16:07:29 fedora steam[12704]: g_object_unref: assertion 'G_IS_OBJECT (object)' failed
Jul 01 16:07:29 fedora com.valvesoftware.Steam.desktop[12704]: Could not connect to X session manager: None of the authentication protocols specified are supported
Jul 01 16:07:29 fedora com.valvesoftware.Steam.desktop[12704]: Could not connect to X session manager: None of the authentication protocols specified are supported
Jul 01 16:07:29 fedora /usr/libexec/gdm-x-session[10138]: (--) NVIDIA(GPU-0): CRT-0: disconnected
...
Jul 01 16:07:29 fedora com.valvesoftware.Steam.desktop[12704]: Could not connect to X session manager: None of the authentication protocols specified are supported
Jul 01 16:07:30 fedora com.valvesoftware.Steam.desktop[12704]: Could not connect to X session manager: None of the authentication protocols specified are supported
Jul 01 16:07:30 fedora com.valvesoftware.Steam.desktop[12704]: Could not connect to X session manager: None of the authentication protocols specified are supported
Jul 01 16:07:30 fedora com.valvesoftware.Steam.desktop[12704]: Could not connect to X session manager: None of the authentication protocols specified are supported
Jul 01 16:07:30 fedora com.valvesoftware.Steam.desktop[12704]: System startup time: 35.18 seconds
Jul 01 16:07:30 fedora steam[12704]: Trying to remove a child that doesn't believe we're it's parent.
Jul 01 16:07:30 fedora steam[12704]: Trying to remove a child that doesn't believe we're it's parent.
Jul 01 16:07:30 fedora steam[12704]: Trying to remove a child that doesn't believe we're it's parent.
Jul 01 16:07:30 fedora steam[12704]: Trying to remove a child that doesn't believe we're it's parent.
Jul 01 16:07:30 fedora steam[12704]: g_object_ref: assertion 'G_IS_OBJECT (object)' failed
Jul 01 16:07:30 fedora steam[12704]: g_object_unref: assertion 'G_IS_OBJECT (object)' failed
Jul 01 16:07:30 fedora com.valvesoftware.Steam.desktop[12704]: BuildCompleteAppOverviewChange: 358 apps
Jul 01 16:07:30 fedora com.valvesoftware.Steam.desktop[12704]: RegisterForAppOverview 1: 12ms
Jul 01 16:07:30 fedora com.valvesoftware.Steam.desktop[12704]: RegisterForAppOverview 2: 12ms
Jul 01 16:07:31 fedora steam[12704]: g_object_ref: assertion 'G_IS_OBJECT (object)' failed
Jul 01 16:07:31 fedora steam[12704]: g_object_unref: assertion 'G_IS_OBJECT (object)' failed
```

**Comment by Zatura** [comment]
I did 

> I found what fixes the issue for me!
> 
>     1. Remove these:
>        
>        * `~/.local/share/Steam/`
>        * `~/.steam/`
>        * `~/.steampath`
>        * `~/.steampid`
> 
>     2. Open Steam.
> 
>     3. Login.
> 
>     4. Go to Steam -> Settings -> Interface.
> 
>     5. Disable "Enable GPU accelerated rendering in web views (requires restart)".
> 
> 
> Never buying NVIDIA again...

For those using flatpak, I removed those folders:
~/.var/app/com.valvesoftware.Steam/.steam
~/.var/app/com.valvesoftware.Steam/.steampath
~/.var/app/com.valvesoftware.Steam/.steampid
~/.var/app/com.valvesoftware.Steam/data/Steam # Not sure about this one, if that's the same one.

Steam manage to run, however i got this tacktrace on journalctl:

```
Jul 01 16:39:47 fedora systemd-coredump[26646]: [🡕] Process 26018 (steamwebhelper) of user 1000 dumped core.
                                                
                                                Stack trace of thread 5862:
                                                #0  0x00007fda4ed6490d n/a (/usr/lib/x86_64-linux-gnu/libc.so.6 + 0x16490d)
                                                #1  0x00007fda34cb3f4e n/a (/usr/lib/x86_64-linux-gnu/GL/default/lib/dri/libgallium_dri.so + 0xb3f4e)
                                                #2  0x00000000004ab4a6 n/a (/home/main/.var/app/com.valvesoftware.Steam/.local/share/Steam/ubuntu12_64/steamwebhelper + 0xab4a6)
                                                ELF object binary architecture: AMD x86-64
```

**Comment by MyCookie** [comment]
Chiming in to add: following https://github.com/ValveSoftware/steam-for-linux/issues/9764#issuecomment-1614911342 solved the issue for me today. I presume that the only thing required is clearing ```.cache/nvidia/GLCache```, AFAIK Electron does not use Vulkan for hardware acceleration on Nvidia.

```xdg-portal-*``` was not a factor in my case, I had ```xdg-portal-kde``` installed and installing or removing any variants did not solve, or worsen the issue for me.

**Comment by Betaminos** [comment]
I am facing the same issue, running Steam (beta build on July 8 2023 - but stable as of late June was similar) on linux kernel 6.4.2, XFCE (xfdesktop is version 4.18.1) and nvidia-dkms (535.54.03).
Starting Steam:
* via command line shows it looping endlessly 
* via command line with "stea -vgui" makes it load and work but friends network is broken (login fails with "Steam friends network unreachable")
* via start menu shows the icon in the task bar but no window ever comes up and cannot be summoned via (e.g.) right-click - Library

> I found what fixes the issue for me!
> 
>     1. Remove these:
>        
>        * `~/.local/share/Steam/`
>        * `~/.steam/`
>        * `~/.steampath`
>        * `~/.steampid`
> 
>     2. Open Steam.
> 
>     3. Login.
> 
>     4. Go to Steam -> Settings -> Interface.
> 
>     5. Disable "Enable GPU accelerated rendering in web views (requires restart)".

Doing this (especially removing ~/.local/share/Steam/) works once.
The moment Steam restarts to change any settings, it breaks down again.
Point (5) does not change this behavior, as I have toggled it on - ran into the issue - and started gain, made sure it is off - and ran into the issue gain.


> Can confirm, deleting `~/.cache/nvidia/GLCache` fixed the issue for me.

This does nothing on my end.
After doing it once, following through with the other steps outlined above to get into Steam, it seems that this folder is not even re-populated on my device.

**Comment by willswats** [comment]
Deleting `~/.var/app/com.valvesoftware.Steam/.cache/nvidia/GLCache/` seems to have worked for me as I'm using the flatpak version of Steam.

**Comment by RAD750** [comment]
This also happens to me with an AMD card. None of the above fixes did the trick, unfortunately.

However launching Steam via the terminal (with just `steam`, not `steam --reset` did the trick).
In the logs, a lot of `Could not connect to X session manager: None of the authentication protocols specified are supported` are spammed.

Specs: AMD Radeon RX480, Pop!_OS 22.04

**Comment by ragardner** [comment]
> This also happens to me with an AMD card. None of the above fixes did the trick, unfortunately.
> 
> However launching Steam via the terminal (with just `steam`, not `steam --reset` did the trick). In the logs, a lot of `Could not connect to X session manager: None of the authentication protocols specified are supported` are spammed.
> 
> Specs: AMD Radeon RX480, Pop!_OS 22.04

I also found that right clicking the Steam icon and clicking "Library" to start steam in the library worked for me, also amd gpu and pop os

**Comment by RAD750** [comment]
Can confirm @ragardner 's workaround works.

**Comment by jorgicio** [comment]
> I found what fixes the issue for me!
> 
> 1. Remove these:
>    
>    * `~/.local/share/Steam/`
>    * `~/.steam/`
>    * `~/.steampath`
>    * `~/.steampid`
> 2. Open Steam.
> 3. Login.
> 4. Go to Steam -> Settings -> Interface.
> 5. Disable "Enable GPU accelerated rendering in web views (requires restart)".
> 
> Never buying NVIDIA again...

Rarely I had GPU accelerated rendering in the morning, after I came back it stopped working. However, there's no need to delete anything.
There's an easier workaround:
1 - Run Steam with steam -vgui.
2 - Disable GPU accelerated rendering in web views, the same way as the same 5 mentioned in the quote.
3 - It asks you to restart Steam, so do it.
4 - After that, close Steam again and then open it with the new UI.
5 - Profit!

**Comment by jorgicio** [comment]
> > I found what fixes the issue for me!
> > 
> > 1. Remove these:
> >    
> >    * `~/.local/share/Steam/`
> >    * `~/.steam/`
> >    * `~/.steampath`
> >    * `~/.steampid`
> > 2. Open Steam.
> > 3. Login.
> > 4. Go to Steam -> Settings -> Interface.
> > 5. Disable "Enable GPU accelerated rendering in web views (requires restart)".
> > 
> > Never buying NVIDIA again...
> 
> Rarely I had GPU accelerated rendering in the morning, after I came back it stopped working. However, there's no need to delete anything. There's an easier workaround: 1 - Run Steam with steam -vgui. 2 - Disable GPU accelerated rendering in web views, the same way as the same 5 mentioned in the quote. 3 - It asks you to restart Steam, so do it. 4 - After that, close Steam again and then open it with the new UI. 5 - Profit!

**Comment by DreamingCuttlefish** [comment]
Issue is still present on laptop with amd GPU

**Comment by murphcid** [comment]
I have tried it on the new Thelio Mira with the AMD 7700x, AMD 6600, and 32gb of RAM, POP!_OS 22.04 with all updates installed, Steam attempts to load and then goes into an endless boot loop, requiring a restart to kill the process.  It also did the same thing in Mint 21.2.  Works great in Windows, no issues at all.  So why now in Linux?  I have tried it from the repo, flatpak and direct from Steam, and same issue.  Got it working once, and then it crashed hard after an hour back to endless boot loops.
